src/tensor_ops/mod.rs

```diff
@@ -252,9 +252,17 @@ pub(crate) use to_dtype::ToDtypeKernel;
 #[cfg(feature = "nightly")]
 mod conv2d;
 #[cfg(feature = "nightly")]
-pub use conv2d::TryConv2D;
+pub use conv2d::{TryConv2D, TryConv2DTo};
+
+#[cfg(feature = "nightly")]
+mod convtrans2d;
 #[cfg(feature = "nightly")]
-pub(crate) use conv2d::TryConv2DTo;
+pub use convtrans2d::{ConvTransAlgebra, TryConvTrans2D, TryConvTrans2DTo};
+
+mod upscale2d;
+pub use upscale2d::{
+    Bilinear, ConstUpscale2D, NearestNeighbor, TryUpscale2D, Upscale2DKernel, UpscaleMethod,
```

Comment:
Don't want to export kernels 😁 
```suggestion
    Bilinear, ConstUpscale2D, NearestNeighbor, TryUpscale2D, UpscaleMethod,
```